src/causalprog/distribution/normal.py

```diff
@@ -76,7 +78,7 @@ def __init__(self) -> None:
         """Create a family of normal distributions."""
         super().__init__(Normal, family_name="Normal")
 
-    def construct(self, mean: ArrayCompatible, cov: ArrayCompatible) -> Normal:
+    def construct(self, mean: ArrayCompatible, cov: ArrayCompatible) -> Normal:  # type: ignore # noqa: PGH003
```

Comment:
mypy was very unhappy with everything I tried putting here. @willGraham01: Any idea what it should be?

Reply:
Also, we originally had this as the following, but either ruff or mypy was not happy that the base class didn't have the `*,`, but having `*, **parameters` in the base class is a syntax error...

```suggestion
    def construct(self, *, mean: ArrayCompatible, cov: ArrayCompatible) -> Normal:  # type: ignore # noqa: PGH003
```

Reply:
I think this is `mypy` expects any overwritten methods to retain a signature that is compatible with the signature in the base class. Personally I would have also thought that `*, mean, cov` was compatible with `**parameters` (since they're all keyword arguments) but in hindsight I can see why it's not (since `**parameters` does not necessitate ANY arguments, but `*, mean, cov` necessitates 2).

I think (for now), leave things as they are (`*, mean, cov`). I can tackle this when I re-work the `DistributionFamily`s using the backend-agnostic classes (which should make this awkward hand-me-down of parameters redundant).